src/App.tsx

```diff
@@ -1,9 +1,14 @@
-import React from 'react';
+import React from "react";
 
 function App() {
   return (
-    <div> 
-      <h1>Hello World</h1>
+    <div className="container">
+      <div className="bg-one"></div>
+      <div className="bg-two">
+        <div className="footer">
+          <p>Este sorteio é meramente ilustrativo e não possui nenhuma ligação com a CAIXA.</p>
+        </div>
+      </div>
     </div>
   );
 }
```

Comment:
This is not wrong but Is not a good practice to put everything into the `App.tsx` file. I imagine you'll refactor it later, but I think you should start to think about it to keep things well organized.